universal-login-relayer/lib/http/routes/authorisation.ts

```diff
@@ -17,13 +22,30 @@ const getPending = (authorisationService : AuthorisationService) =>
     return responseOf({ response: result });
   };
 
-const denyRequest = (authorisationService : AuthorisationService) =>
-  async (data: {walletContractAddress: string, body: {key: string}}) => {
+const denyRequest = (authorisationService : AuthorisationService, provider: providers.Provider) =>
+  async (data: {walletContractAddress: string, body: {key: string, signature: utils.Signature}}) => {
+    const {walletContractAddress} = data;
+    const {key, signature} = data.body;
+
+    const cancelAuthorisationRequest: CancelAuthorisationRequest = {walletContractAddress, key};
+    const [isValid, computedAddress] = verifyCancelAuthorisationRequest(cancelAuthorisationRequest, signature, key);
+    if (!isValid) {
```

Comment:
You can replace it with `ensure(...)`

Reply:
You can actually create an `ensureCancelAuthorisationRequestSignature` and push ensure to this function